app/models/piece.rb

```diff
@@ -4,6 +4,7 @@ class Piece < ApplicationRecord
   def move!(x, y)
     return false unless valid_move?(x, y)
     update_attributes(x_position: new_x, y_position: new_y)
+    Move.create(piece_id: id)
```

Comment:
This should be commented-out until the `Move` model exists, otherwise it will cause an error for all the other branches.